cypress/e2e/visual-regression.cy.js

```diff
@@ -37,15 +37,13 @@ describe('Visual regression tests ', function() {
 			cy.visit('/apps/files')
 		})
 	})
-	after(function() {
-		cy.logout()
-	})
 
 	it('See files in the list', function() {
-		cy.get('.files-fileList tr[data-file="test-card.mp4"]', { timeout: 10000 })
-			.should('contain', 'test-card.mp4')
-		cy.get('.files-fileList tr[data-file="test-card.png"]', { timeout: 10000 })
-			.should('contain', 'test-card.png')
+		// TODO: Do we care about the file name being split by a ' ' in the text property?
+		cy.getFile('test-card.mp4', { timeout: 10000 })
+			.should('contain', 'test-card .mp4')
```

Comment:
With the new version the file name and the extension exist in different spans. It looks like this leads to a space being inserted between them in the text content of the parent tag.

So far i adjusted the tests to match this behavior. But maybe we want to prevent it as well?

Reply:
Yeah, just as a clarification, this is a security measure for some specific file names where you can trick the extension into being wrongly displayed and have `.exe` shown as `.pdf`